### PR TITLE
add clitools

### PR DESCRIPTION
**Pull request by vhouchouas**
Allow to execute some commands from the CLI, such as:
- `php clitools.php install`;
- `php clitools.php checkAgenda`;
- `php clitools.php database create`;
- `php clitools.php api checktokens`;
- etc.

### CODE REVIEW

File: DBAgenda.php [hunk 1]
```diff
@@ -22,6 +22,55 @@ public function __construct(string $CalDAV_url, string $CalDAV_username, string
...
+                $this->log->info("Category $category[name] will be deleted.");
+            }
+        }
+        $query = $this->pdo->prepare("DELETE " . $sql);
+        $query->execute();
+        $this->log->info("Cleaning orphan categories - done.");
+    }
+    
+    public function clean_orphan_attendees($quiet = true) {
+        $sql = "FROM attendees WHERE not exists (
```
Comment (gturri) [inline DBAgenda.php:46]: for the same reason as `clean_orphan_categories` I think this should be in a transaction (to ensure we print exactly what we delete)

**Review by gturri — COMMENTED** [review]
This looks good to me!
I would just be thrilled to understand why we can remove the `USE <database>` statement before merging it (the other comments are rather details)

File: MySQLAgenda.php [hunk 2]
```diff
@@ -30,8 +30,7 @@ protected function openDB() {
     }
     
     public function createDB() {
-        $this->pdo->query("USE {$this->db_name};");
```
[the remark this answers is not in the document]
Comment (vhouchouas) [reply MySQLAgenda.php:33]: After a `DROP DATABASE db;` + `CREATE DATABASE db;` we must call `USE db;` to select it.
As we open the required database on L23 `MySQLAgenda.php` we can drop it.

[the remark this answers is not in the document]
Comment (gturri) [reply MySQLAgenda.php:33]: Ah, yes, of course!
Thanks for the explanation.

**Pull request merged**